### PR TITLE
fix: Removed SAML references from SCIM

### PR DESCRIPTION
**Pull request by infinitetrooper**
## Checklist
I have:
- [x] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [ ] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [ ] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [ ] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/advanced-concepts/user-provisioning-group-sync.mdx [hunk 1]
```diff
@@ -48,9 +48,9 @@ This page shows how to configure Appsmith for SCIM to enable the automatic provi
 To complete the setup, go to your IdP application that is connected to Appsmith and follow the steps below:
 
 <Tabs queryString="scim">
-<TabItem label="Active Directory (SAML)" value="scim-ad-saml">
+<TabItem label="Active Directory" value="scim-ad">
 
-1. On the Active Directory homepage, click **Enterprise application** on the sidebar and open the SAML application that is already connected to Appsmith.
+1. On the Active Directory homepage, click **Enterprise application** on the sidebar and open the application that is already connected to Appsmith.
```
Comment (jnikhila) [inline website/docs/advanced-concepts/user-provisioning-group-sync.mdx:53]: Does the application have to be connected to Appsmith?

Comment (infinitetrooper) [reply website/docs/advanced-concepts/user-provisioning-group-sync.mdx:53]: If they configured SSO first (which is most likely), then yes.

Comment (jnikhila) [reply website/docs/advanced-concepts/user-provisioning-group-sync.mdx:53]: What if they don't, still SCIM works right? We should mention that if you have configured SSO, use the same connected app or create a new app on the provider.

Comment (infinitetrooper) [reply website/docs/advanced-concepts/user-provisioning-group-sync.mdx:53]: SCIM connection works but, SSO won't so, it makes little sense to setup SCIM before SSO.

**Review by jnikhila — APPROVED** [review]

**Pull request merged**